src/macros.rs

```diff
@@ -329,51 +325,54 @@ macro_rules! service {
             rpc $fn_name( $( $arg : $in_ ),* ) -> $out | $error;
         }
     };
-    // Pattern for when all return types have been expanded
+// Pattern for when all return types have been expanded
```

Comment:
It looks like a new version hasn't been released in like 2 months? Not sure why.